tests/view_tests/templates/jsi18n.html

```diff
@@ -4,6 +4,14 @@
 </head>
 
 <body>
+  <p id="formats">
```

Comment:
I think it's nice to non-selenium test as much as possible since selenium tests aren't run as part of the normal build. If you think the selenium test is also adding value, it's okay with me to keep it.